dakara_server/library/test_feeder.py

```diff
@@ -0,0 +1,53 @@
+from django.core.urlresolvers import reverse
+from django.contrib.auth import get_user_model
+from rest_framework import status
+
+from .base_test import BaseAPITestCase
+
+
+UserModel = get_user_model()
+
+
+class FeederListViewTestCase(BaseAPITestCase):
+    url = reverse("library-feeder-list")
+
+    def setUp(self):
+        # create a user without any rights
+        self.user = self.create_user("TestUser")
+
+        # create a manager
+        self.manager = self.create_user("TestManager", library_level=UserModel.MANAGER)
+
+        # create test data
+        self.create_library_test_data()
+
+    def test_get_feeder_song_list(self):
+        """Test to get feeder song list for manager
+        """
+        # Login as simple user
+        self.authenticate(self.manager)
+
+        # Get songs list
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data), 2)
+
+        # Songs are sorted by title
```

Comment:
Songs are not supposed to be sorted at all. There is no `order_by` in `FeederListView.queryset`.